contracts/ethereum/contracts/Registry.sol

```diff
@@ -131,6 +121,17 @@ contract Registry is IRegistry, ITotalRewardPool {
         return cumulative;
     }
 
+    function operator() public view returns (address) {
+        return operator_;
+    }
+
+    function updateOperator(address _newOperator) public {
+        require(msg.sender == operator(), "only operator");
+        require(_newOperator != address(0), "zero operator");
```

Comment:
Registry depends on Operator explicitly, so it needs to have it supported, since the DAO will be operating this function, it's better to have it baked in explicitly imo